lambda/custom-erc721/scripts/custom-erc721-upgradeable-scripts/deployProxy.js

```diff
@@ -0,0 +1,69 @@
+// Script to deploy Smart contract using Metakeep Lambda and REST API.
+import fs from "fs";
+import env from "dotenv";
+import { create, getLambdaAddress } from "../../../lambdaUtils.mjs";
+import getDeveloperWallet, {
+  checkAPIKey,
+  waitUntilTransactionMined,
+} from "../../../../helpers/utils.mjs";
+import Web3 from "web3";
+
+async function main() {
+  env.config();
+
+  const web3 = new Web3();
+
+  // Checks if the API_KEY is set in the .env file.
+  checkAPIKey();
+
+  const data = JSON.parse(
+    fs.readFileSync(
+      "../smart-contracts/artifacts/contracts/CustomERC721Proxy.sol/CustomERC721Proxy.json"
+    )
+  );
+
+  const developerAddress = await getDeveloperWallet();
+
+  // call initialize using data field of proxy contract
+
+  const initalizationParameters = [
+    "MetaKeep Originals",
+    "MTKP",
+    developerAddress,
+  ];
+
+  const functionSignature = "initialize(string,string,address)";
+
+  const functionSelector =
+    web3.eth.abi.encodeFunctionSignature(functionSignature);
+
+  console.log(functionSelector);
+
+  const encodedParameters = web3.eth.abi.encodeParameters(
+    ["string", "string", "address"],
+    initalizationParameters
+  );
+
+  const initializationData = functionSelector + encodedParameters.slice(2);
```

Comment:
why not use `encodeFunctionSignature ` https://web3js.readthedocs.io/en/v1.2.11/web3-eth-abi.html#id11

Reply:
done